net/zerotier/files/etc/init.d/zerotier

```diff
@@ -41,6 +41,7 @@ start_service() {
 
 	if [ ${enabled} -eq 0 ]; then
 		echo "disabled in /etc/config/zerotier"
+		return 1
```

Comment:
I think the best thing to do here is to simply put a plain `return` instead of a `return 1` ([as I had put here](https://github.com/mwarning/zerotier-openwrt/pull/128/files#diff-500e5f9436fc26df6e76fbff4e9e3abf7b95b6de567fe91764888e3e2877e5b1R45)), I don't think it is necessary for the startup script to return an error if the user has ZeroTier disabled by configuration.

Reply:
done